internal/cloud/tfe_client_mock.go

```diff
@@ -1090,21 +1098,22 @@ func newMockWorkspaces(client *MockClient) *MockWorkspaces {
 	}
 }
 
-func (m *MockWorkspaces) List(ctx context.Context, organization string, options tfe.WorkspaceListOptions) (*tfe.WorkspaceList, error) {
+func (m *MockWorkspaces) List(ctx context.Context, organization string, options *tfe.WorkspaceListOptions) (*tfe.WorkspaceList, error) {
 	wl := &tfe.WorkspaceList{}
-
 	// Get all the workspaces that match the Search value
 	searchValue := ""
-	if options.Search != nil {
-		searchValue = *options.Search
-	}
-
 	var ws []*tfe.Workspace
 	var tags []string
 
-	if options.Tags != nil {
-		tags = strings.Split(*options.Tags, ",")
+	if options != nil {
```

Comment:
Could you add some context for this change?

Reply:
I see - so it also looks like we're looking for search and tag options now, whereas we weren't doing that before?

Reply:
Right! So, in the past those two fields, tags and Search, were pointer to strings, so rather than checking that they were not empty strings, we were able to do `if options.Tags != nil {` and `if options.Search != nil {`, and then feel safe using their values if their values were not nil. 
But since Search and Tags fields are no longer pointer to strings, but just strings, that means we need go back to the old fashioned way of making sure they are not empty strings `if len(options.Search) > 0 {` and `if len(options.Tags) > 0 {` before we use their values.

Reply:
Got it - thank you for the explanation!